docs/docs-content/integrations/konvoy.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_label: "Konvoy"
+title: "Konvoy"
+description: "Konvoy Kubernetes distribution"
+hide_table_of_contents: true
+type: "integration"
+category: ["kubernetes", "amd64"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: "https://registry.spectrocloud.com/v1/k8s-dashboard/blobs/sha256:2de5d88b2573af42d4cc269dff75744c4174ce47cbbeed5445e51a2edd8b7429?type=image/png"
+tags: ["packs", "konvoy", "kubernetes"]
+---
+
+[Konvoy](https://d2iq.com/products/konvoy) is a Kubernetes distribution by D2iQ. Konvoy is built on pure open-source
+Kubernetes and includes a full suite of integrated and supported best-of-breed Day 2 platform applications for an
+out-of-the-box, production-ready experience.
+
+Palette offers Konvoy as an available Kubernetes distribution for creating and managing clusters. You can use Konvoy
+when creating a [cluster profile](../profiles/cluster-profiles/create-cluster-profiles/create-cluster-profiles.md) and
+then use it to create a cluster.
+
+## Versions Supported
+
+<Tabs queryString="versions">
+
+<TabItem label="1.27.x" value="1.27.x">
+
+## Prerequisites
+
+- A minimum of 4 CPU and 4 GB Memory.
+
+- Users or groups mapped to a Kubernetes RBAC role.
+
+- Konvoy requires the [BYOOS](./byoos.md) pack to reference the image created through the Konvoy image builder project.
+  Refer to the [Usage](#usage) section for more information.
+
+- Operating System (OS) dependencies as listed in the table.
+
+### Supported Infrastructure Platforms
+
+Palette supports the following infrastructure platforms and Operating Systems (OS) combinations for Konvoy.
+
+| Infrastructure Platform | OS                       | Supported?         |
+| ----------------------- | ------------------------ | ------------------ |
+| VMware vSphere          | Red Hat Linux Enterprise | :white_check_mark: |
```

Comment:
Do we need this table while it has 1 row?

Reply:
Yeah, we should comment out the table for now. More versions and combinations will be added but you're right.